easybuild/easyconfigs/s/Spyder/Spyder-5.0.5-foss-2020b.eb

```diff
@@ -0,0 +1,278 @@
+easyblock = 'PythonBundle'
+
+name = 'Spyder'
+version = '5.0.5'
+
+homepage = 'https://github.com/spyder-ide/spyder'
+description = """Spyder is an interactive Python development environment providing MATLAB-like features in a simple
+ and light-weighted software."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'cstd': 'c++11'}
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+dependencies = [
+    ('Python', '3.8.6'),
+    ('ZeroMQ', '4.3.3'),
+    ('PyQt5', '5.15.1'),
+    ('IPython', '7.18.1'),
+    ('geopandas', '0.9.0'),
+    ('libspatialindex', '1.9.3'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+# strip out (too strict version) requirements for pyqt5 and pyqtwebengine,
+# because they are not installed with pip, but are provided via the PyQt5 dependency
+local_spyder_preinstallopts = (
+    """sed -i "s/'pyqt5.*,//g" setup.py && """
+    """sed -i "s/'pyqtwebengine.*,//g" setup.py && """
+)
+
+exts_list = [
+    ('astroid', '2.6.5', {
+        'checksums': ['83e494b02d75d07d4e347b27c066fd791c0c74fc96c613d1ea3de0c82c48168f'],
+    }),
+    ('async_generator', '1.10', {
+        'checksums': ['6ebb3d106c12920aaae42ccb6f787ef5eefdcdd166ea3d628fa8476abe712144'],
+    }),
+    ('autopep8', '1.5.7', {
+        'checksums': ['276ced7e9e3cb22e5d7c14748384a5cf5d9002257c0ed50c0e075b68011bb6d0'],
+    }),
+    ('binaryornot', '0.4.4', {
+        'checksums': ['359501dfc9d40632edc9fac890e19542db1a287bbcfa58175b66658392018061'],
+    }),
+    ('bleach', '3.3.1', {
```

Comment:
Already included in deps (slightly different version, but maybe still compatible)